api/src/services/fields.ts

```diff
@@ -418,7 +418,12 @@ export class FieldsService {
 		if (field.schema?.has_auto_increment) {
 			column = table.increments(field.field);
 		} else if (field.schema?.data_type) {
-			column = table.specificType(field.field, field.schema.data_type);
+			// Append length to data type if it's stored in schema
+			const data_type =
+				field.schema?.max_length && field.schema.data_type !== 'longtext'
```

Comment:
Instead of hardcoding `longtext` here, lets figure out why it returns the length in the first place and fix it there 🙂

Reply:
Would prefer that too! 😄 
But I don't have any clue where to look at for this...

Reply:
Riiiiiiiiight here:

https://github.com/knex/knex-schema-inspector/blob/49f1e107bf7402fbe4516f1b976de56f6ba39e88/lib/dialects/mysql.ts#L58

Reply:
Niiiice, thanks 😄

Reply:
This is no longer an issue, as max_length isn't used for `text` types 💪🏻

Reply:
Nice 💪🏻